include/vara/Feature/Location.h

```diff
@@ -0,0 +1,69 @@
+#ifndef VARA_FEATURE_LOCATION_H
+#define VARA_FEATURE_LOCATION_H
+
+#if defined(STD_EXPERIMENTAL_FILESYSTEM)
+#warning "using std::experimental::filesystem"
+
```

Comment:
```suggestion
```
In the llvm style headers do not separate the parts inside a ifdef with extra lines